osf/metrics/reports.py

```diff
@@ -37,7 +37,7 @@ class Meta:
 
 class YearmonthField(metrics.Date):
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs, format='strict_year_month', required=True)
+        super().__init__(*args, **kwargs, format='strict_year_month')
```

Comment:
Minor: This makes a lot of `YearMonth` fields no longer implicitly (sort of) required. Would it be preferable to default to `required=True` but allow `required=False` to be passed in? e.g. with `required = kwargs.pop('required', True)`

Reply:
the default for `elasticsearch_dsl.Field` is `required=False`, and consistency with that seems like a good idea -- there was only one yearmonth field prior to this project (on the `MonthlyReport` base class), which now has explicit `required=True`, and the ones added for this project seem fine to make optional